src/metax_api/api/rest/base/serializers/file_serializer.py

```diff
@@ -97,6 +97,11 @@ def is_valid(self, raise_exception=False):
             self.initial_data['parent_directory'] = self._get_id_from_related_object(
                 'parent_directory', self._get_parent_directory_relation)
 
+        data = self.initial_data
```

Comment:
I made this to separate variable because it was impossible to get flake8 to accept the long sentence :I so dunno if this was an improvement now at all...

Reply:
had to google how this should take care of with flake8 and the internet answered

Reply:
Thanks! Now it should be...